dss/events/handlers/index.py

```diff
@@ -79,18 +90,18 @@ def create_index_data(blobstore, bucket_name, bundle_id, manifest, logger):
                 continue
             try:
                 file_key = create_blob_key(file_info)
-                file_string = blobstore.get(bucket_name, file_key).decode("utf-8")
+                file_string = handle.get(bucket_name, file_key).decode("utf-8")
                 file_json = json.loads(file_string)
             # TODO (mbaumann) Are there other JSON-related exceptions that should be checked below?
             except json.decoder.JSONDecodeError as ex:
                 logger.warning(f"In bundle {bundle_id} the file \"{file_info[BundleFileMetadata.NAME]}\""
```

Comment:
this needs to be fixed as per #427 (separate diff though, pls)

Reply:
will do in future pr